sdk-extensions/autoconfigure/src/testOtlp/java/io/opentelemetry/sdk/autoconfigure/OtlpConfigTest.java

```diff
@@ -133,14 +133,14 @@ void configureExportersGeneral() {
     props.put("otel.exporter.otlp.endpoint", "https://localhost:" + server.httpsPort());
     props.put("otel.exporter.otlp.certificate", certificate.certificateFile().getAbsolutePath());
     props.put("otel.exporter.otlp.headers", "header-key=header-value");
-    props.put("otel.exporter.otlp.timeout", "5s");
+    props.put("otel.exporter.otlp.timeout", "15s");
     ConfigProperties properties = ConfigProperties.createForTest(props);
     SpanExporter spanExporter = SpanExporterConfiguration.configureExporter("otlp", properties);
     MetricExporter metricExporter =
         MetricExporterConfiguration.configureOtlpMetrics(
             properties, SdkMeterProvider.builder().build());
 
-    assertThat(spanExporter).extracting("timeoutNanos").isEqualTo(TimeUnit.SECONDS.toNanos(5));
+    assertThat(spanExporter).extracting("timeoutNanos").isEqualTo(TimeUnit.SECONDS.toNanos(15));
```

Comment:
You'll want to update `.join(10, TimeUnit.SECONDS)` to match the new timeout.

Reply:
@anuraaga ?

Reply:
My intention was to make sure the test still has a "real" timeout of 10s like most of our tests, the 15 is mostly to check the value is respected, while we still think the test should pass fast. If it's confusing I could raise the join timeout though.